utils/createTables.js

```diff
@@ -1,5 +1,5 @@
 var manager = require("./db/manager");
 
-manager.createTable("requests")
+manager.ensureTablesExist()
   .then(manager.closeConnection)
   .then(process.exit);
```

Comment:
We should probably change this to:

` .then(() => manager.closeConnection)`

otherwise the promise from `createTable()` in  `ensureTablesExisits` will get passed into `closeConnection` and it will pass through the `if/then`. If we do this, we probably don't need to explicitly call `process.exit`

Reply:
Yep, thanks.